docs/faction-design.md

```diff
@@ -0,0 +1,219 @@
+
+## About
+
+The point of this document is to outline the server's current and future intended design.
+
+The current design section should be updated by contributors of pull requests which significantly affect balance or add new features. This will serve as a useful overview for new players and code contributors to understand the existing differences between the factions and a high-level overview of gameplay.
+
+The future design section should be mainly updated by maintainers as a central location for their vision for the server. This gives direction for code contributors to build their PRs on and gives players insight into what to look forward to.
+
+The main USP of TGMC is: “asymmetrical team deathmatch PvP gameplay with a high degree of QoL”.
+
+## Current Design
+
+### Visual
+
+- Xenomorphs - arachnid/dinosaur mix.
+- TG Marines - highly flamboyant armour design and accessorising similar to Destiny, or dressed in modern military attire if preferred. Ballistic, incendiary and explosive weaponry with a modern design, as well as futuristic laser weaponry.
+- Sons of Mars - orange mirrored helmets, dark orange accent colour on black armoured suits. High-tech volkite weaponry.
+
+### Gameplay
+
+The two main factions Xenomorphs and TG Marines have assymetrical combat. This means whilst there may be a few distinct exceptions in their gameplay, they each have unique strengths, weaknesses and other differences.
+
+#### Xenomorphs
+
+Xenos have 4 tiers of playable castes. Tier 1-3 have decreasing amounts of availability to players as they move up the tiers. The tiers more or less increase the combat prowess of the player with increasing tiers. They also serve to funnel some players into the weaker roles that they typically wouldn't play by having a player cap based off of total players on the team. Tier 4 allows only one playable per caste and typically comes with minimum player requirements.
+
+Xenos gain maturity over time, progressing from Young to Mature to Elder to Ancient to Primordial. At each stage their stats improve, and for some castes their abilities improve as well e.g. Widows have a greater max amount of spiderlings.
```

Comment:
that part will be removed later if the xeno maturity removal PR goes out (i think)

Reply:
Yeah this will be changed if those PRs get merged before this, otherwise it makes more sense for those PRs to update this doc too.